components/azure/azuredevops/queue-pipeline/component.yaml

```diff
@@ -0,0 +1,29 @@
+name: Queue Azure Pipeline
+description: |
+  A Kubeflow pipeline component to queue an Azure Pipeline.
+
+inputs:
+    - {name: organization, type: String, description: 'Azure DevOps organization'}
+    - {name: project, type: String, description: 'Azure DevOps project'}
+    - {name: id, type: Integer, description: 'Azure Pipeline definition id'}
+    - {name: pat_env, type: String, default: '', description: 'Name of environment variable containing Azure DevOps PAT'}
+    - {name: pat_path_env, type: String, default: '', description: 'Name of environment variable containing path to Azure DevOps PAT'}
+    - {name: source_branch, type: String, default: '', description: 'Source branch for the pipeline'}
+    - {name: source_version, type: String, default: '', description: 'Source version for the pipeline'}
+    - {name: parameters, type: String, default: '', description: 'Parameters for the pipeline'}
+outputs:
+    - {name: url, description: 'Url of the queued pipeline'}
+implementation:
+    container:
+        image: ''
```

Comment:
Is it possible to specify a real public image here?

Reply:
Sorry, unfortunately, we cannot have a public image for now